docs/notebooks/measurement-error-mitigation.ipynb

```diff
@@ -0,0 +1,576 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "source": [
+    "# Measurement Error Mitigation on Quantum Inspire\n",
+    "Pieter Eendebak <pieter.eendebak@tno.nl><br/>\n",
+    "\n",
+    "For more details see [Qiskit textbook on measurement error mitigation](https://qiskit.org/textbook/ch-quantum-hardware/measurement-error-mitigation.html) and [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/readme.html).\n",
+    "\n",
+    "For quantum devices, in particular spin-qubits, the measurement error is significant with respect to other sources of errors. We can reduce the effect of measurement errors using measurement error mitigation. This is particularly important when different qubits have different readout fidelities or when there is a cross-talk between the qubits readouts.\n",
+    "\n",
+    "To install the required dependencies for this notebook run: `pip install qiskit rich quantuminspire requests`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "import numpy as np\n",
+    "import qiskit\n",
+    "import requests\n",
+    "from qiskit import Aer, QuantumCircuit, QuantumRegister, assemble, execute, transpile\n",
+    "from qiskit.ignis.mitigation.measurement import CompleteMeasFitter, complete_meas_cal\n",
+    "from qiskit.providers.aer.noise import NoiseModel, ReadoutError\n",
+    "from qiskit.providers.aer.noise.errors import depolarizing_error, pauli_error\n",
+    "from qiskit.visualization import plot_histogram\n",
+    "from quantuminspire.api import QuantumInspireAPI\n",
+    "from quantuminspire.credentials import get_token_authentication, load_account\n",
+    "from quantuminspire.qiskit import QI\n",
+    "from rich import pretty\n",
+    "\n",
+    "pretty.install()\n",
+    "\n",
+    "\n",
+    "def get_qi_calibration_from_qiskit_result(qi_job, api, authentication) -> dict:\n",
+    "    \"\"\" Return calibration data from a QI job \"\"\"\n",
+    "    project = api.get_project(qi_job.job_id())\n",
+    "    # NOTE: due to limited capabilties of the QI framework this is the latest job, perhaps not the actual job\n",
+    "    job = api.get_jobs_from_project(project[\"id\"])[0]\n",
```

Comment:
You don't need to get project first and ask the jobs from the project. Use:
job = qi_job.get_jobs()[0]